app/views/surveys/component/_side-slider.pug

```diff
@@ -2,10 +2,9 @@ aside.record-list.side-slider
 	button.btn-icon-only.close(type="button", aria-hidden="true") ×
 	h3(data-i18n='record-list.title')= t('record-list.title')
 	p(data-i18n='record-list.msg1')= t('record-list.msg1')
-	progress.record-list__upload-progress
```

Comment:
It turns out we already had a separate progress bar on top of the 'side slider' that shows the records. I think it didn't have much utility because of the feedback already provided when succeeding or failing to upload records (animation when removing a record, and showing error messages below failing record). I think this gives sufficient feedback on the progress.